dist/react-widgets.js

```diff
@@ -1370,10 +1370,9 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	    var elementProps = omit(this.props, Object.keys(propTypes));
 	    var listProps = pick(this.props, Object.keys(List.propTypes));
-	    var popupProps = pick(this.props, Object.keys(_Popup2['default'].propTypes));
-
 	    var PopupComponent = tetherPopup ? _TetheredPopup2['default'] : _Popup2['default'];
 
+	    var popupProps = pick(this.props, Object.keys(PopupComponent.propTypes));
```

Comment:
I believe this should be `popUp` 

I only make this point as below we have `dropDown`

I don't care either way, just consistency, and considering dropDown is already there, we should probably go with that.